README.md

```diff
@@ -1012,6 +1012,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
 
 *See also [Task management/To-do lists](#task-managementto-do-lists) and [Project Management](#project-management)*
 
+  * [Brimir](http://getbrimir.com/) - Simple and clean opens-oruce ticket manager written in Ruby on Rails. `AGPLv3` `Ruby`
```

Comment:
Open-source is misspelled.

Please add source code link: https://github.com/ivaldi/brimir